community/sc-im/depends

```diff
@@ -0,0 +1,2 @@
+bison
```

Comment:
`bison make`? Is bison needed at runtime or only build time?

Reply:
`bison make` it's the answer.